pontoon/base/static/js/main.js

```diff
@@ -137,11 +137,26 @@ var Pontoon = (function (my) {
         '</li>');
 
         // Sort by quality
-        var listitems = ul.children("li");
+        var listitems = ul.children("li"),
+          sourceMap = {
+            'Translation memory': 1,
+            'Mozilla': 2,
+            'Open Source': 3,
+            'Microsoft': 4,
+            'Machine Translation': 5
```

Comment:
You could add additional 2-space indent to this entire block, so that `listitems` and `sourceMap` are aligned.